core/tests/webdriverio_utils/ClassroomPage.js

```diff
@@ -40,6 +42,15 @@ var ClassroomPage = function() {
       expect(topicSummaryTiles.length).toEqual(0);
     }
   };
+
+  this.launchDiagnosticTestPage = async function() {
+    await waitFor.elementToBeClickable(
+      launchDiagnosticTestPageButton,
+      'Launch diagnostic test button taking longer than expected'
+    );
```

Comment:
You shouldn't need this. `action.click` includes a `waitFor.elementToBeClickable()` call already

Reply:
Removed the unnecessary checks. Thank you for the guidance.